routing_model/src/scenario_tests.rs

```diff
@@ -825,6 +825,110 @@ mod dst_tests {
             &AssertState::default(),
         );
     }
+
+    #[test]
+    fn test_local_event_offline() {
+        run_test(
+            "Get local event node detected offline",
+            &intial_state_old_elders(),
+            &[LocalEvent::NodeDetectedOffline(NODE_ELDER_130).to_event()],
+            &AssertState {
+                action_our_votes: vec![ParsecVote::Offline(NODE_ELDER_130)],
+                ..AssertState::default()
+            },
+        );
+    }
+
+    #[test]
+    fn test_local_event_online_again() {
```

Comment:
Is this test non-sensical, or potentially exposing missing checks in the flows? Getting a BackOnline local event shouldn't happen here right?

Reply:
It really depend whether we trust our event provider. If it was an RPC, I'd say we need to handle it gracefully. This one is a bit controversial, regardless, I think this test make sense, as we would not want to filter it at this stage, if we wanted to do something, that would be when it comes out of Parsec.

The reason is that when this event occur we are not in shared state, so we could not trust that all nodes take the same action.